src/utils/reshapedmsbuild.fs

```diff
@@ -784,7 +792,15 @@ module internal ToolLocationHelper =
 //        instance.GetType().GetPropserty(propName, BindingFlags.Public).SetValue(instance, propValue, null)
 
     type ResolveAssemblyReference () =
-        let assembly = Assembly.Load(new AssemblyName("Microsoft.Build.Tasks.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))
+        let assembly =
+            Assembly.Load(
+                new AssemblyName(
+#if !NETSTANDARD1_5
```

Comment:
fsproj doesnt compile for netstandard, so it's ok i think.
dotnet sdk (project.json) by default add the normalized define of the framework name under build (`netstandard1.6` => `NETSTANDARD1_6`, 'net46` => `NET46`). that's usefull, i think new msbuild target (after project.json) is going to have that too by default